docs/contracts/abstract-contracts.rst

```diff
@@ -6,27 +6,31 @@
 Abstract Contracts
 ******************
 
-Contracts are marked as abstract when at least one of their functions lacks an implementation as in the following example (note that the function declaration header is terminated by ``;``)::
+Contracts need to be marked as abstract when at least one of their functions was not implemented.
+This can be done by using the ``abstract`` keyword as shown in the following example. Note, that this contract need to be
+defined as abstract, because the function ``utterance()`` was defined, but no implementation was provided (no implementation body ``{ }`` was given).::
 
     pragma solidity >=0.4.0 <0.7.0;
 
-    contract Feline {
+    abstract contract Feline {
         function utterance() public returns (bytes32);
     }
 
-Such contracts cannot be compiled (even if they contain implemented functions alongside non-implemented functions), but they can be used as base contracts::
+Such abstract contracts can only be used as base contracts. This is also true, if an abstract contract itself does implement
```

Comment:
I'm not sure what you mean by "only be used as base contracts". They can also be used to contain function definitions, define types, etc.

The main thing that can not be done with abstract contracts is creating them directly with the `new` operator and requesting their bytecode from the compiler.